### ISSUE TITLE
Doesn't compile on odin dev-2022-11

### ISSUE THREAD

**Issue by alexgb0** [DanielGavin/ols#163]
# Build output
```
/home/alex/.local/bin/ols/src/common/allocator.odin(69:2) Unhandled switch case: Alloc_Non_Zeroed
        Suggestion: Was '#partial switch' wanted?
```

# Odin version
```
Odin: dev-2022-11:df19c48d
OS:   Arch Linux, Linux 5.19.9-arch1-1
CPU:  Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz
RAM:  15748 MiB
```

## Notes
Can build successfully with `#partial switch`, but it seems it breaks it because every time modify a file giving an Error
```
Error: Language client is not ready yet when handling textDocument/didChange
```

**Comment by DanielGavin** [comment]
It seems Odin was just changed recently. You can use https://github.com/DanielGavin/ols/releases/tag/nightly it the meantime.

**Comment by DanielGavin** [comment]
Fixed in b66acc5e24611001b07ed73445b851f54769809a
